app/assets/javascripts/index/layers/data.js

```diff
@@ -86,6 +86,16 @@ OSM.initializeDataLayer = function (map) {
 
     if (dataLoader) dataLoader.abort();
 
+    let spanLoading = $("#layers-data-loading");
+
+    if (!spanLoading.length) {
+      spanLoading = $("<span>")
+        .attr("id", "layers-data-loading")
+        .appendTo($("#label-layers-data"));
+
+      spanLoading.append(" " + I18n.t("javascripts.map.layers.loading"));
```

Comment:
I am not sure if this "append" is safe in laguages which are written from right to left.

Reply:
This is used in other places for loading indicators:
https://getbootstrap.com/docs/5.3/components/spinners/

Reply:
@AntonKhorev Thanks for the recommendation. I replaced a text with a bootstrap spinner.

Reply:
With rtl the spinner should be at the left.

https://developer.mozilla.org/en-US/docs/Web/CSS/:dir
Could be a solution with ::after or ::before elements.

Reply:
@HolgerJeromin Now it works this way when Arabic is chosen as a default language, spinner is shown on the left side of the text.

https://github.com/user-attachments/assets/21d7c1ef-8bec-4664-9236-7ecc5aef5b81

Reply:
I have no idea how it works, but the video looks nice :-)

Reply:
`Loading` text was changed with the loader spinner.